README.md

```diff
@@ -0,0 +1,4 @@
+>>  To install the project and run the test
```

Comment:
You can style this file much better using markdown. Please check it out.

Reply:
Ok Boss